### DISCUSSION TITLE
StatisticsPen's slant value

### DISCUSSION THREAD

**Discussion by miguelsousa** [fonttools/fonttools#3222] [Q&A] [answered]
@behdad Is there a relationship between the [`slant` value computed by the StatisticsPen](https://github.com/fonttools/fonttools/blob/3f46e67e8b63b8720816bf9cbdcf2e602e1b1806/Lib/fontTools/pens/statisticsPen.py#L70-L71) and a glyph's italic angle defined in degrees?

Asking because the `slant` value of the **uppercase i** of [SourceSans3-It.otf](https://github.com/adobe-fonts/source-sans/blob/release/OTF/SourceSans3-It.otf) is 0.199695. Using a drawing app I can see that the glyph's italic angle is about 11 degrees.

**Comment by behdad** [comment, marked as answer]
```
In [3]: math.tan(math.radians(11))
Out[3]: 0.19438030913771848

In [5]: math.degrees(math.atan(0.199695))
Out[5]: 11.293128399694282
```

**Reply by miguelsousa** [reply]
Thank you!
